metricbeat/docker-compose.yml

```diff
@@ -21,6 +24,8 @@ beat:
     - MYSQL_HOST=mysql
     - MYSQL_PORT=3306
     - POSTGRESQL_DSN=postgres://postgres@postgresql:5432?sslmode=disable
+    - POSTGRESQL_HOST=postgresql
+    - POSTGRESQL_PORT=5432
```

Comment:
For the docker entrypoint script below. Otherwise the beat will not wait for postgres to properly start.